### DISCUSSION TITLE
What will happen if I Synced some dotfile but then i decided to ignore it?

### DISCUSSION THREAD

**Discussion by DravenJohnson** [lra/mackup#1974] [Q&A]
Will this software put it back? or I just lost it?

**Comment by dirkdirk** [comment]
It helps to know what's really going on to fix this issue, so read this: [what it really does](https://github.com/lra/mackup#bullsht-what-does-it-really-do-to-my-files)

Now you understand that macup moves the dot files to the storage of your choice, then creates a sym link to said dot file.

To manually remove a specific app after it's already been mackup'd.
Add the app name to ignore list (supported apps listed on home page) with $ vim /.mackup.cfg
Open iCloud Drive in Finder.
Find where app's files are kept in /Mackup.
Copy them back to local machine in the right places.
Run: $ mackup backup, answer y at the prompts.
Look on iCould Drive to ensure the app's files aren't there.

This process worked for Atom for me (I'm using Atom package Sync-settings package).

If anyone sees/knows of any shortcoming of this method, please let us know!

**Comment by josegus** [comment]
> It helps to know what's really going on to fix this issue, so read this: [what it really does](https://github.com/lra/mackup#bullsht-what-does-it-really-do-to-my-files)
> 
> Now you understand that macup moves the dot files to the storage of your choice, then creates a sym link to said dot file.
> 
> To manually remove a specific app after it's already been mackup'd. Add the app name to ignore list (supported apps listed on home page) with $ vim /.mackup.cfg Open iCloud Drive in Finder. Find where app's files are kept in /Mackup. Copy them back to local machine in the right places. Run: $ mackup backup, answer y at the prompts. Look on iCould Drive to ensure the app's files aren't there.
> 
> This process worked for Atom for me (I'm using Atom package Sync-settings package).
> 
> If anyone sees/knows of any shortcoming of this method, please let us know!

So you have to do all the steps manually just to remove an app?
- remove dotfiles from your computer
- remove from Storage Cloud
- remove all symlinks previously created
... and try to not break anything
